### ISSUE TITLE
It is difficult to see the CLI widget and the access token

### ISSUE THREAD

**Issue by dimitar-peychinov** [semaphoreci/docs#1435]
On the CLI documentation https://docs.semaphoreci.com/reference/sem-command-line-tool/ it is stated that

> You can find the exact sem connect command, which includes your current organization's name and access token, by selecting the CLI widget in the top-right corner of any screen on Semaphore.
> 
> Your access token is always available, and revokable, on your [account page](https://me.semaphoreci.com/account).
> 

So there are two suggestions how to find the access token.
I tried both but none of them worked. Here are the details:

### 1. The CLI widget seems to be missing
Here's what I see:
<img width="404" alt="Screenshot 2024-05-20 at 14 23 09" src="https://github.com/semaphoreci/docs/assets/169891028/00f1a279-4aee-4e62-a780-0ae031b732c9">
And here is what I expected to see (source: https://semaphoreci.medium.com/ci-cd-for-microservices-on-digitalocean-kubernetes-b913fa0b3404):

![code_icon](https://github.com/semaphoreci/docs/assets/169891028/1cff6441-2808-4376-adb1-9aeb1a1e08d7)

The code icon is not visible (at least for me).

### 2. I can not see my access token on my account/profile page.
Here's what I see:
<img width="562" alt="Screenshot 2024-05-20 at 14 30 55" src="https://github.com/semaphoreci/docs/assets/169891028/9bf93d78-7449-4e01-b301-420f41a25e68">
If the API token is the `access token` that I need, then I can't copy/get it from here either.

I assume that my organisation settings or my account's permissions in the organisation may have something to do with this - but if that's the reason then it would be nice to have a note about it in the documentation.

### Update:
Later today I saw CLI widget:
<img width="570" alt="Screenshot 2024-05-20 at 14 52 09" src="https://github.com/semaphoreci/docs/assets/169891028/8463533a-515d-4dec-83ed-dae6e90d42ac">

And I thought "Here's the access token now!" - but to my disappointment the "full command" only stated
`sem connect studitemps.semaphoreci.com _YOUR_API_TOKEN_` :( 

So, to further add some details to the questions/issues above:
1. The CLI widget is not shown "on any screen" in Semaphore - just on some, and I can't tell on which ones.
2. This is not "top right" (as stated in the documentation) - it's more like top-left-after-some-titles. :) 
3. The connect command does not (anymore?) include the actual token - but a placeholder instead - _YOUR_API_TOKEN_
<img width="570" alt="Screenshot 2024-05-20 at 15 07 19" src="https://github.com/semaphoreci/docs/assets/169891028/6183c0d8-e73c-4e8d-a73b-2449713cc2e5">

I assume that's a security improvement, but let's not leave the documentation behind. :)

**Comment by radwo** [comment]
Hi Dimitar,

Thank you for reaching out!

Some time ago, to tighten the security of the API token, we made it readable only once during its creation. If you lose it, you will need to regenerate it. Sorry for the inconvenience.

Once again, thank you for the feedback. We'll make sure to update the documentation as needed.

**Comment by radwo** [comment]
I merged the changes to the documentation and will close this ticket now. Feel free to reopen it if you have any concerns.

**Comment by radwo** [comment]
You are right. The quick fix here didn't do the trick. I will revisit that documentation during the week. Thanks for pointing this out.
